16.md

```diff
@@ -11,13 +11,18 @@ Relays may decide to allow replaceable and/or ephemeral events.
 Replaceable Events
 ------------------
 A *replaceable event* is defined as an event with a kind `10000 <= n < 20000`.
-Upon a replaceable event with a newer timestamp than the currently known latest replaceable event with the same kind, the old event SHOULD be discarded and replaced with the newer event.
+Upon a replaceable event with a newer timestamp than the currently known latest replaceable event with the same kind and author, the old event SHOULD be discarded and replaced with the newer event.
 
 Ephemeral Events
 ----------------
 An *ephemeral event* is defined as an event with a kind `20000 <= n < 30000`.
 Upon an ephemeral event being received, the relay SHOULD send it to all clients with a matching filter, and MUST NOT store it.
 
+Specifically Replaceable Events
+------------------
+A *specific replaceable event* is defined as an event with a kind `30000 <= n < 40000`.
+Upon a replaceable event with a newer timestamp than the currently known latest replaceable event with the same kind, author, and the first tag (and its values), the old event SHOULD be discarded and replaced with the newer event.
```

Comment:
Should it say "its value" instead as everywhere we only deal with the first value of tags, i.e. the second element in the array?

Reply:
Agree, but I wouldn't make this match the first tag, I would mark the tag similarly to how we do it in nip 10 or use a tag called `deduplication`. Clients have had problems in the past because of ordering of tags had a meaning but was not consistent.

Reply:
I agree with disregarding anything beyond the second element of a tag.

@Cameri the "lets use the first tag" was born from the idea to use tags that already have a meaning otherwise. In the chat it was about e-tags to define properties of group chats with that e-tag ID. I agree with you that it would be desirable to use something else. `unique` over `deduplication` though.